doc/source/science_guide/sg_boundary_forcing.rst

```diff
@@ -120,7 +120,12 @@ parameterizations. Rain and all melted snow end up in the ocean.
 Wind stress and transfer coefficients for the
 turbulent heat fluxes are computed in subroutine
 *atmo\_boundary\_layer* following :cite:`Kauffman02`, with additions and changes as detailed in Appendix A of :cite:`Roberts15` for high frequency coupling (namelist variable ``highfreq``).
-The resulting equations are provided here.
+The resulting equations are provided here for the default boundary layer 
+scheme, which is based on Monin-Obukhov theory (``atmbndy`` = ‘stability’). 
+Alternatively, ``atmbndy`` = ‘constant’ provides constant coefficients for
+wind stress, sensible heat and latent heat calculations (computed in subroutine
+*atmo\_boundary\_const*); ``atmbndy`` = ‘mixed’ uses the stability based 
+calculation for wind stress and constant coefficients for sensible and latent heat fluxes.
```

Comment:
Small formatting nit: I would avoid these "fancy" single quotes as copy pasting them directly to the namelist is asking for trouble :P Maybe just put the whole thing inside the double backticks, like this:


```suggestion
The resulting equations are provided here for the default boundary layer 
scheme, which is based on Monin-Obukhov theory (``atmbndy = 'stability'``). 
Alternatively, ``atmbndy = 'constant'`` provides constant coefficients for
wind stress, sensible heat and latent heat calculations (computed in subroutine
*atmo\_boundary\_const*); ``atmbndy = 'mixed'`` uses the stability based 
calculation for wind stress and constant coefficients for sensible and latent heat fluxes.
```